### ISSUE TITLE
Crash when accessing a Craft Manager or Crafting Grid

### ISSUE THREAD

**Issue by smpwojcik** [NillerMedDild/Enigmatica6#1642]
Issue doesn't seem to be present on other mod packs with Refined Storage. If it is unrelated to the Enigmatica 6 mod pack, let me know and I'll try to get it to the Refined Storage team instead. 

Crafting Grid: https://pastebin.com/S3cAs08K
Craft Manager: https://pastebin.com/2bYjsRwQ

**Comment by NillerMedDild** [comment]
Thanks for the report. Which version of the modpack are you using? Please follow the bug template :)

It would be wise to update your Java fyi, [here's a guide on how to do it for the CurseForge launcher](https://gist.github.com/NillerMedDild/40b4992b7a3e5e653a353fe1d45f2185#Java)

**Comment by smpwojcik** [comment]
I am using version 0.4.3.

Thanks for the java info. Sorry about not using the template. It was my first time reporting a bug and I wasn't aware. I will look for it next time.

**Comment by NillerMedDild** [comment]
Ah no worries, welcome to GitHub 😄 Could you supply screenshots of the setup please? I haven't been able to reproduce the issue.

**Comment by smpwojcik** [comment]
I've attempted to reproduce the crash again, but I haven't had luck either. I've since updated my java, so I'm wondering if the old version of java that I was running was the cause? 

I'm assuming by "setup", you're refering to the layout of the blocks. 

![2021-03-12_12 01 16](https://user-images.githubusercontent.com/2474900/110980611-a4ffec00-832b-11eb-93c0-40d98a435d85.png)
![2021-03-12_12 01 29](https://user-images.githubusercontent.com/2474900/110980617-a6311900-832b-11eb-9966-c3f1d4565e0c.png)
![2021-03-12_12 01 48](https://user-images.githubusercontent.com/2474900/110980619-a6c9af80-832b-11eb-9f02-e92f8adc8430.png)
![2021-03-12_12 01 56](https://user-images.githubusercontent.com/2474900/110980622-a7624600-832b-11eb-83c1-c62e6146c4ae.png)
